wiki/telescope_design.md

```diff
@@ -114,7 +114,9 @@ can also use a laptop. We use a pi because it is cheaper and easier to maintain.
 We provide a ready to run SD card image which can get you started.
 
 1. Download the latest [CHART Pi image](https://galileo.sese.asu.edu/chart/).
+  - If you're ambitious and want to install the software from scratch, you can follow our "Creating a Disk Image" instructions in the [github repo](https://github.com/astrochart/CHART).
```

Comment:
I would give more guidance than "if you're ambitious" how about "those wishing to diverge from the Pi setup will need to install the libraries. This is the path for hackers who want to help us fix bugs, make new features, or install on some other computer besides a Raspberry Pi. "